be/src/storage/rowset/scalar_column_iterator.cpp

```diff
@@ -328,7 +328,7 @@ Status ScalarColumnIterator::get_row_ranges_by_bloom_filter(const std::vector<co
     RETURN_IF(!_reader->has_bloom_filter_index(), Status::OK());
     bool support = false;
     for (const auto* pred : predicates) {
-        support = support | pred->support_bloom_filter();
```

Comment:
`|` maybe used for performance

Reply:
in modern compiler, there is no different in performance. but || can make code more clear